src/relion/zocalo/wrapper.py

```diff
@@ -538,10 +531,10 @@ def _(stage_object: relion.Class3D, job_string: str, relion_options: RelionItOpt
     return ispyb_command_list
 
 
-def ispyb_attachment(file_name, file_path, file_type):
+def ispyb_attachment(attachment_path_object, file_type):
     return {
         "ispyb_command": "add_program_attachment",
-        "file_name": file_name,
-        "file_path": file_path,
+        "file_name": attachment_path_object.name,
+        "file_path": attachment_path_object.parent,
```

Comment:
`file_path` might need an `os.fspath()` wrapper, I don't know how well this is handled during serialisation